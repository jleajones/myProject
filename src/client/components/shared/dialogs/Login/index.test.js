import React from 'react';
import { shallow } from 'enzyme';

import LoginDialog from './index';

describe('<LoginDialog />', () => {
  test('Renders component without crashing', () => {
    const handleCloseDialog = jest.fn();
    const open = false;
    const component = shallow(
      <LoginDialog handleClose={handleCloseDialog} open={open} />
    );

    expect(component).toMatchSnapshot();
  });
});
