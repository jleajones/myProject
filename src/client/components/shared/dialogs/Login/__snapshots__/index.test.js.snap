// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginDialog /> Renders component without crashing 1`] = `
<Dialog
  fullScreen={false}
  label="login-dialog-title"
  onClose={[MockFunction]}
  open={false}
  title="Login"
>
  <WithStyles(ForwardRef(DialogTitle))
    id="login-dialog-title"
  >
    Login
  </WithStyles(ForwardRef(DialogTitle))>
  <e
    accessType="online"
    buttonText="Login"
    cookiePolicy="single_host_origin"
    disabledStyle={
      Object {
        "opacity": 0.6,
      }
    }
    fetchBasicProfile={true}
    icon={true}
    isSignedIn={false}
    jsSrc="https://apis.google.com/js/api.js"
    onFailure={[Function]}
    onRequest={[Function]}
    onSuccess={[Function]}
    prompt=""
    scope="profile email"
    tag="button"
    theme="light"
    type="button"
    uxMode="popup"
  />
</Dialog>
`;
